modules/apps/data-engine/data-engine-taglib/src/main/java/com/liferay/data/engine/taglib/servlet/taglib/util/DataLayoutTaglibUtil.java

```diff
@@ -303,15 +293,8 @@ private DataLayout _getDataLayout(
 		throws Exception {
 
 		DataLayoutResource dataLayoutResource = DataLayoutResource.builder(
-		).endpoint(
-			_portal.getHost(httpServletRequest),
-			httpServletRequest.getServerPort(), httpServletRequest.getScheme()
-		).header(
-			"Cookie",
-			"JSESSIONID=" +
-				CookieKeys.getCookie(httpServletRequest, CookieKeys.JSESSIONID)
-		).parameter(
-			"p_auth", AuthTokenUtil.getToken(httpServletRequest)
+		).user(
+			_portal.getUser(httpServletRequest)
```

Comment:
@ealonso that is how you need to call the resource. Use the resource from rest-api not from client.